docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+services:
+  webapp:
```

Comment:
Totally fine to leave it "webapp",  but just to mention a convention we've used elsewhere is to call it "app"